src/AppInstallerCLI.sln

```diff
@@ -428,6 +430,30 @@ Global
 		{3BAF989F-7F65-465B-ACE8-BAFE42D1017E}.Release|x64.Build.0 = Release|x64
 		{3BAF989F-7F65-465B-ACE8-BAFE42D1017E}.Release|x86.ActiveCfg = Release|x86
 		{3BAF989F-7F65-465B-ACE8-BAFE42D1017E}.Release|x86.Build.0 = Release|x86
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|ARM.ActiveCfg = Debug|ARM
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|ARM.Build.0 = Debug|ARM
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|ARM64.Build.0 = Debug|ARM64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|x64.ActiveCfg = Debug|x64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|x64.Build.0 = Debug|x64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|x86.ActiveCfg = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Debug|x86.Build.0 = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|ARM.ActiveCfg = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|ARM.Build.0 = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|ARM64.ActiveCfg = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|ARM64.Build.0 = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|x64.ActiveCfg = Debug|x64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|x64.Build.0 = Debug|x64
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|x86.ActiveCfg = Debug|Win32
+		{866C3F06-636F-4BE8-BC24-5F86ECC606A1}.Fuzzing|x86.Build.0 = Debug|Win32

```

Comment:
I don't think cpprestsdk will be used in fuzzing tests, fuzzing tests are for yaml fuzzing. can you double check?